### PR TITLE
Allows adjusting jumpsuits through the strip panel

### PR DESCRIPTION
**Pull request by Mickyan**
## About The Pull Request

Adds the option to adjust jumpsuit style from the strip panel, the option only appears if the target is wearing a jumpsuit that can be adjusted

Screenshot_10

## Why It's Good For The Game

It's weird that you can rip the pants off someone but can't use a zipper, this one time I wanted to put a medal on a dead crewmember and I couldn't do it because they had their jumpsuit open and it made me sad

It can be quite useful for medical doctors and paramedics especially since you may sometimes need to treat someone without having to disconnect their internals by stripping them completely

:cl:
tweak: You can now use the strip menu to adjust the jumpsuit style of other people
fix: Humans with robot torsos can have their belt, ID, pockets slots correctly accessed through the strip menu
/:cl:

### CODE REVIEW

**Comment by nemvar** [comment]
slippery slope.

**Comment by Mickyan** [comment]
> slippery slope.

pants

**Comment by cacogen** [comment]
he had that in the chamber

**Comment by Dennok** [comment]
If you here. Please make Belt, Pockets, ID avaliable if target has robochest too.
as you can see here.
https://github.com/tgstation/tgstation/blob/ad3fd3fec23845d3211648f6a6c59438be099983/code/modules/mob/living/carbon/human/species.dm#L869-L874
Someone with robochest can equip Belt, Pockets and ID but no one can remove it (exept for this someone)

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -167,7 +167,12 @@
...
 	else
-		dat += "<tr><td><B>Uniform:</B></td><td><A href='?src=[REF(src)];item=[ITEM_SLOT_ICLOTHING]'>[(w_uniform && !(w_uniform.item_flags & ABSTRACT)) ? w_uniform : "<font color=grey>Empty</font>"]</A></td></tr>"
+		dat += "<tr><td><B>Uniform:</B></td><td><A href='?src=[REF(src)];item=[ITEM_SLOT_ICLOTHING]'>[(w_uniform && !(w_uniform.item_flags & ABSTRACT)) ? w_uniform : "<font color=grey>Empty</font>"]</A>"
+		if(w_uniform)
+			var/obj/item/clothing/under/U = w_uniform
+			if (U.can_adjust)
+				dat += "&nbsp;<A href='?src=[REF(src)];toggle_uniform=[ITEM_SLOT_ICLOTHING]'>Adjust</A>"
+		dat += "</td></tr>"
 
 	if((w_uniform == null && !(dna && dna.species.nojumpsuit)) || (ITEM_SLOT_ICLOTHING in obscured))
```
Comment (Dennok) [inline code/modules/mob/living/carbon/human/human.dm]:
```suggestion
	var/obj/item/bodypart/O = get_bodypart(BODY_ZONE_CHEST) 
	if((w_uniform == null && !(dna && dna.species.nojumpsuit) && !(O && O.status == BODYPART_ROBOTIC)) || (ITEM_SLOT_ICLOTHING in obscured))
```

Comment (Mickyan) [reply code/modules/mob/living/carbon/human/human.dm]: seems to be working, I'll include it

**Comment by Dennok** [comment]
Thanks.

**Comment by cacogen** [comment]
does surgery work with this?

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**